src/man/kiwix-manage.1

```diff
@@ -1,62 +1,72 @@
-.TH KIWIX 1 "21 May 2012"
+.TH KIWIX-MANAGE 1 "21 May 2012"
+
 .SH NAME
 kiwix\-manage \- Kiwix Library Manager
+
 .SH SYNOPSIS
 .IX Header SYNOPSIS
-.B kiwix\-manage
-.br
-kiwix\-manage LIBRARY_PATH add ZIM_PATH ...
-.br
-	kiwix-manage LIBRARY_PATH show [CONTENTID1] [CONTENTID2] ...
-.br
-	kiwix\-manage LIBRARY_PATH remove CONTENTID1 [CONTENTID2]
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH add ZIM_PATH ...
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH show [ZIM_ID_1] [ZIM_ID_2] ...
+.TP
+\fBkiwix\-manage\fR LIBRARY_PATH remove ZIM_ID_1 [ZIM_ID_2] ...
```

Comment:
What about using bold font for the actions (add, show, remove)?

Reply:
yes, sounds a good idea